README.md

```diff
@@ -14,11 +14,14 @@ In the initial version the following utils are provided:
 * ```setup_tracing(name)``` - Sets up basic tracing using  a standardised naming convebstion so that the application is easily identifiable in visualisation tools.
 * ```set_console_exporter()``` - Turns on output of the capturesd traces in a local console/terminal to allow viewing of it without the need for an observability backend such as Jaeger or Promethus. Useful for debugging and testing.
 * ```get_tracer(name)``` - Retrieves the currently active Tracer object and labels is using a standard naming convention so that traces it produces are consistent across applications.
-* ```get_trace_context()``` - Retrives the current trace context (this is just a more clearly named version of the library function).
 * ```get_context_propagator``` - Retrieves the current observability context info in a form suitable for propagating.
 * ```propagate_context_in_stomp_headers(headers, context)``` - Simplfies the propagation of the Tracing Context between services that support STOMP communication over a message bus.
 * ```retrieve_context_from_stomp_headers(frame)``` - Simplifies th reception of the Tracing Context by services that support STOMP communication over a message bus.
 * ```add_span_attributes``` - Simplifies the addition of named attributes to the current span.
+* ```JsonObjectSpanExporter``` - A custom SpanExporter that allows the span content to be examined for use
+in tests.
+* ```asserting_span_exporter``` - A contextmanager that makes use of JsonObjectSpanExporter to allow
```

Comment:
I hadn't noticed this existed, there are ways to auto-generate it, might be worth making an issue for it.

Reply:
will do